src/nestor/handlers/contacts.py

```diff
@@ -223,3 +227,18 @@ def __get_all_contacts(self) -> str:
         
         return Colorizer.highlight("\n".join([str(record) for record in self.book.data.values()]))
     
+    @input_error({IndexError: "Contact name is required"})
```

Comment:
`{IndexError: "Contact name is required"}` is defaulted in the `input_error`, so don't need to specify it here